.nvmrc

```diff
@@ -0,0 +1 @@
+lts/hydrogen
```

Comment:
we should use the same version being used in the dockerfile.
e.g.: fix it at the current version (`npm install lts/hydrogen; npm use lst/hydrogen; node -v`)
- It's not a good idea do use `latest` images on docker, since a broken image may be pushed to the repo, so it should be slowly updated/tested.

Reply:
this is a third party lib, we don't use docker here, but it's a good idea to use 18 (LTS) on our other repos